src/python/pants/core/util_rules/partitions.py

```diff
@@ -133,7 +140,7 @@ def _single_partition_field_sets_partitioner_rules(cls) -> Iterable:
     )
     async def partitioner(
         request: _PartitionFieldSetsRequestBase, subsystem: SkippableSubsystem
-    ) -> Partitions:
+    ) -> Partitions[Any, FieldSet]:
```

Comment:
I think `Any` is wrong now, it should be `None`.

Reply:
That doesn't work because `None` doesn't implement the `PartitionKey` protocol 😞